include/grpc/support/alloc.h

```diff
@@ -32,6 +32,9 @@ typedef struct gpr_allocation_functions {
   void* (*zalloc_fn)(size_t size); /** if NULL, uses malloc_fn then memset */
   void* (*realloc_fn)(void* ptr, size_t size);
   void (*free_fn)(void* ptr);
+  void* (*aligned_alloc_fn)(size_t size, size_t alignment);
+  void* (*cacheline_alloc_fn)(size_t size);
```

Comment:
What is the significance of `cacheline_alloc`? That basically is `aligned_alloc_fn(CACHE_LINE)`?

Reply:
Removed.